pkg/k8s/apis/cilium.io/utils/utils.go

```diff
@@ -54,16 +58,24 @@ var (
 // GetPolicyLabels returns a LabelArray for the given namespace and name.
 func GetPolicyLabels(ns, name string, uid types.UID, derivedFrom string) labels.LabelArray {
 	// Keep labels sorted by the key.
-	return labels.LabelArray{
+	labelsArr := labels.LabelArray{
 		labels.NewLabel(k8sConst.PolicyLabelDerivedFrom, derivedFrom, labels.LabelSourceK8s),
 		labels.NewLabel(k8sConst.PolicyLabelName, name, labels.LabelSourceK8s),
-		labels.NewLabel(k8sConst.PolicyLabelNamespace, ns, labels.LabelSourceK8s),
-		labels.NewLabel(k8sConst.PolicyLabelUID, string(uid), labels.LabelSourceK8s),
 	}
+
+	if ns != "" {
+		labelsArr = append(labelsArr, labels.NewLabel(k8sConst.PolicyLabelNamespace, ns, labels.LabelSourceK8s))
```

Comment:
to avoid long lines
```golang
nsLabel := labels.NewLabel(k8sConst.PolicyLabelNamespace, ns, labels.LabelSourceK8s)
labelsArr = append(labelsArr, nsLabel)
```